packages/ember-glimmer/tests/integration/content-test.js

```diff
@@ -484,6 +484,30 @@ class DynamicContentTest extends RenderingTest {
     this.assertContent('hello');
     this.assertInvariants();
   }
+
+  ['@test it can render a property on a function']() {
+    let func = () => {};
+    func.aProp = 'this is a property on a function';
+
+    this.renderPath('func.aProp', { func });
+
+    this.assertContent('this is a property on a function');
+
+    this.assertStableRerender();
+
+    this.runTask(() => set(func, 'aProp', 'still a property on a function'));
```

Comment:
this is a new feature, the regression was rendering a prop on a function not using Ember.set() to update a prop on a function. https://ember-twiddle.com/c35628b179d00989b7469f84f1d08558?openFiles=controllers.application.js%2C I tried several versions of ember, props on functions were rendered as constant.